tests/sles4sap/hana_install.pm

```diff
@@ -16,6 +16,7 @@ use strict;
 use warnings;
 use testapi;
 use utils qw(file_content_replace zypper_call);
+use Utils::Systemd qw(systemctl);
```

Comment:
```suggestion
use Utils::Systemd 'systemctl';
```

Reply:
Really? It was "forbidden" previously!

Reply:
> Really? It was "forbidden" previously!

That's what I had understood, however I cannot find either the PR or the issue where it was discussed. I did check this repo and os-autoinst and the number of uses with the single quotes is like 30 times more, so I'll apply the suggestiong from @ricardobranco777

Reply:
Found the PR: https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/9931

Reply:
Ok, this explains why the last PRs where I pointed this had no issue with Travis. Thx!